client/css/widgets/scoreboard.css

```diff
@@ -0,0 +1,44 @@
+.scoreboard {
+  box-sizing: border-box;
+  background: #ffffff;
+  color: #ffffff;
+  border: 2px solid black;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.scoreboard table {
+  padding: none;
+  border:  none;
+  border-collapse: collapse;
+  display: block;
+  overflow-x: scroll;
+  overflow-y: scroll;
+}
+
```

Comment:
Add the following so that as you scroll right the top-left cell is in front of the other .firstRow cells.
```css
.firstRow.firstCol {
  z-index: 2;
}
```